### ISSUE TITLE
Hydrostatic balance

### ISSUE THREAD

**Issue by byphilipp** [openclimatefix/graph_weather#18]
Your forecasting system used 13 levels instead <=5 in others similar systems

The hydrostatic balance equation can greatly improve your system.

You could add to the loss function the hydrostatic balance discrepancy in each of the layers:
( (Tv_i+Tv_{i+1})/2 + g/R * (H_{i+1}-H_{i}) / log(p_{i+1}/p_i) )^2 -> min
where 
g = 9.8065 m/s^2 is the gravitational acceleration
R = 287.04 is gas constant for air
Tv [K] = T [K] *(1+Q [kg/kg]/(Q + R/Rv)) is virtual temperature
Rv = 461.51 is gas constant for vapor

**Comment by jacobbieker** [comment]
Thanks for the suggestion! Yeah, I can try adding that. I'm hoping to train these models on even more levels and variables than in the paper, and at a finer resolution.

**Comment by peterdudfield** [comment]
@all-contributors please add @byphilipp  for ideas
